### PR TITLE
Custom Speech: add en-US structured text sample

### PR DESCRIPTION
**Pull request by ArcherAZ**
## Purpose

* ...

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ X ] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

**Comment by ArcherAZ** [comment]
@BrianMouncer Could you please help review and approve? :) (This is for CLM public preview samples)

File: sampledata/customspeech/en-US/training/structured-text.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+- python
+- java
+- .net
+
+
+@ speech:phoneticlexicon
+- .net/D AO T N EH T
+
+# SomeTrainingSentence
+- Microsoft custom speech supports structured text input files in {@locale}
```
Comment (mahilleb-msft) [inline sampledata/customspeech/en-US/training/structured-text.md:17]: I would suggest to capitalize Custom Speech, as we do in the docs (also next two lines)

File: sampledata/customspeech/en-US/training/structured-text.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+- english
+- spanish
+- french
+- german
+
+@ list programming_language =
+- python
+- java
+- .net
+
```
Comment (mahilleb-msft) [inline sampledata/customspeech/en-US/training/structured-text.md:11]: consider dropping this empty line

**Review by andreasjl — APPROVED** [review]
:shipit:

File: sampledata/customspeech/en-US/training/structured-text.md (hunk 1 above)
Comment (mahilleb-msft) [reply sampledata/customspeech/en-US/training/structured-text.md:17]: (assuming that case does not matter here)

**Review by mahilleb-msft — APPROVED** [review]

**Comment by ArcherAZ** [comment]
Could you help merge this to PROD?

**Comment by ArcherAZ** [comment]
@andreasjl i saw this PR has been approved for a month but haven't merged (i'm not authorized). Would you proceed?

**Comment by andreasjl** [comment]
@ArcherAZ I am not authorized, either. Maybe @mahilleb-msft can merge it?

**Review by trrwilson — APPROVED** [review]

**Pull request merged**